tests/src/test/java/alluxio/client/fs/LoadMetadataIntegrationTest.java

```diff
@@ -214,6 +214,16 @@ public void syncOverrideLoadMetadata() throws Exception {
     checkGetStatus("/mnt/dir1/dirA/file", options, true, true, 1);
   }
 
+  @Test
+  public void loadMetadataListDir() throws Exception {
+    ListStatusPOptions listOptions =
+        ListStatusPOptions.newBuilder().setRecursive(false)
+            .setLoadMetadataType(LoadMetadataPType.ONCE)
+            .setCommonOptions(FileSystemMasterCommonPOptions.newBuilder().setSyncIntervalMs(-1)
+            ).build();
+    checkListStatus("/mnt/", listOptions, true, true, 3);
```

Comment:
Could you add a comment on what the "undesired" behavior would have done? I think it would have performed more UFS operations, right? How many more?